modules/ztp-worker-node-daemon-selector-compatibility.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+// Epic CNF-5335 (4.11), Story TELCODOCS-643
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-additional-worker-daemon-selector-comp"]
```

Comment:
_{context}  Also, for whatever reason, this module isn't appearing in the navigation links on the right side of the preview screen even though it renders properly.